### PR TITLE
CB-16767  Added request logging to SaltConnector before sending the r…

### PR DESCRIPTION
**Pull request by SubhanHakverdiyev**
…equest

See detailed description in the commit message.

### CODE REVIEW

**Review by doktoric — APPROVED** [review]

File: orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/client/SaltConnector.java [hunk 1]
```diff
@@ -133,6 +133,7 @@ public GenericResponse health() {
     @Measure(SaltConnector.class)
     @Retryable(value = ClusterProxyWebApplicationException.class, backoff = @Backoff(delay = 1000))
     public GenericResponses pillar(Iterable<String> targets, Pillar pillar) {
+        LOGGER.debug("Executing salt pillar targets: {}, pillar: {}", targets, pillar);
```
Comment (keyki) [inline orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/client/SaltConnector.java:136]: Do you have an example log line? Pillars could have sensitive data in them like DB passwords.

Comment (SubhanHakverdiyev) [reply orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/client/SaltConnector.java:136]: Unfortunately I don't have example log line but I agree they can contain sensitive data in this case do you suggest to keep just `targets` in log and eliminating `pillar`?

Comment (SubhanHakverdiyev) [reply orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/client/SaltConnector.java]: @keyki  I have eliminated the pillar.

Comment (keyki) [reply orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/client/SaltConnector.java]: Thanks.

**Pull request merged**